packages/webpack/index.js

```diff
@@ -17,18 +17,45 @@ if (!requiredRoot || !~requiredRoot.indexOf('webpack.config.js')) {
   requiredRoot = process.cwd();
 }
 
-/**
- * WebPack wrapper for the asset-bundle module.
- *
- * @constructor
- * @param {String} filename The location of where the asset should be written.
- * @param {Object} options asset-bundle options.
- * @public
- */
 class WebPack {
-  constructor(filename, options) {
-    this.options = options;
-    this.name = filename;
+  constructor(name = 'bundle.svgs', options = {}) {
+    const { plugins, namespace, bundler, modify, root } = options;
+
+    this.name = name;     // Name of the file that needs to be written to disk.
+    this.output = '';     // Output directly that we receive from webpack.
+
+    //
+    // Create the asset-bundle, but instead of passing it with paths
+    // we need to pass it an empty array. We're going to call the processing
+    // steps our selfs as we don't have the assets yet.
```

Comment:
spelling nit: that should be `ourselves`

Reply:
Done, thanks for the review.